bot.js

```diff
@@ -40,32 +42,61 @@ mongoose.connect(
 mongoose.Promise = global.Promise;
 
 
-
-
 //Execute Commands
-bot.on('message', message => {
-  if(message.author.equals(bot.user)) return;
-  if(!message.content.startsWith(PREFIX)) return;
+bot.on("message",function(message) {
```

Comment:
let's use Arrow functions
```suggestion
bot.on("message", message => {
```

Reply:
done